### PR TITLE
Nozomi fix lost incidents

### PR DESCRIPTION
**Pull request by nicoloereni**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.py [hunk 1]
```diff
@@ -6,25 +6,46 @@
...
+        self.auth = auth or (
+            demisto.params().get("credentials", {}).get('identifier', ''),
+            demisto.params().get("credentials", {}).get('password', '')
         )
-        return response
+
+    def _make_request(self, method, path, **kwargs):
+        url = self.base_url + path
+        demisto.info("Nozomi url " + url)
+        response = requests.request(
```
Comment (YairGlik) [inline Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.py:28]: It is not recommended to use requests.request,
It is safer to use _http_request from CommonServerPython.
To determine the status code they are ok You can send in the status_code parameter.

**Review by YairGlik — CHANGES_REQUESTED** [review]
Good job!

**Comment by YairGlik** [comment]
Hi @nicoloereni,
We're ready for a demo. Please let me know when you're available for one over DFIR.
We'll talk about the http_request in the meeting.

**Comment by nicoloereni** [comment]
> Hi @nicoloereni, We're ready for a demo. Please let me know when you're available for one over DFIR. We'll talk about the http_request in the meeting.

Hi @YairGlik, perfect!
I'm in the CET timezone, I'm sending my availability via Slack.
Thanks

**Review by YairGlik — APPROVED** [review]

**Comment by nicoloereni** [comment]
@YairGlik 
What can I do to make the `demisto/pull-check` geen?

**Comment by nicoloereni** [comment]
@YairGlik 
All checks are green, if you want you can merge it, thanks

**Comment by Shellyber** [comment]
For the Reviewer: Trigger build request has been accepted for this contribution PR.

**Comment by Shellyber** [comment]
For the Reviewer: Successfully created a pipeline in GitLab with url: https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1757182

**Comment by Shellyber** [comment]
For the Reviewer: Trigger build request has been accepted for this contribution PR.

**Comment by Shellyber** [comment]
For the Reviewer: Successfully created a pipeline in GitLab with url: https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1759405

**Comment by nicoloereni** [comment]
@YairGlik @Shellyber 

Our common customer has reached out to request an ETA for the fix of this issue. They are looking forward to having this resolved soon.
Thank you for your support!

**Comment by YairGlik** [comment]
Hello,

Thank you for your contribution to the project! There are a few small fixes needed to get your changes ready for merging:

1. Docker Image: The current image version (demisto/python3:3.10.14.92207) is outdated. Please update it to the latest version: demisto/python3:3.11.10.116949.
2. Description File: A description file is missing for the integration. Please add a file named NozomiNetworks_description.md and include the necessary description.
If you need any help or further clarification, feel free to reach out.

Thank you and good luck!

**Comment by Shellyber** [comment]
For the Reviewer: Trigger build request has been accepted for this contribution PR.

**Comment by Shellyber** [comment]
For the Reviewer: Successfully created a pipeline in GitLab with url: https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1777902

**Pull request merged**